lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb

```diff
@@ -483,6 +483,14 @@ def remove_column(table_name, column_name, type = nil, options = {}) #:nodoc:
           clear_table_columns_cache(table_name)
         end
 
+        def remove_columns(table_name, *column_names, type: nil, **options) # :nodoc:
+          quoted_column_names = column_names.map { |column_name| quote_column_name(column_name) }.join(", ")
+
+          execute "ALTER TABLE #{quote_table_name(table_name)} DROP (#{quoted_column_names}) CASCADE CONSTRAINTS"
```

Comment:
What do you think about removing `CASCADE CONSTRAINTS` here? How about executing `remove_foreign_key` explicitly?

Reply:
I've specified the same `CASCADE CONSTRAINS` that `remove_column` method uses.

```ruby
def remove_column(table_name, column_name, type = nil, options = {}) #:nodoc:
  execute "ALTER TABLE #{quote_table_name(table_name)} DROP COLUMN #{quote_column_name(column_name)} CASCADE CONSTRAINTS"
ensure
  clear_table_columns_cache(table_name)
end
```

https://github.com/rsim/oracle-enhanced/pull/2182/files#diff-6392b894ebd6ea18f2983b5b9c77d0b191850a63d62897cd3d64be83ea1f45a1R480-R484

I still don't understand benefits and risks of replacing it with `remove_foreign_key` method.

The aim of this PR is to make `remove_columns` executable with a single SQL statement. Is it possible to separate it as another issue?

Reply:
Oh, right. Let's make this behavior consistent.